interview-questions.md

```diff
@@ -8,36 +8,76 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort and a model called Student. The cohort model has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-Your answer:
+Your answer: 
+To fix this you would go to the model called Student and add the association `belongs_to : Cohort`
 
 Researched answer:
+To create a new instance of a student we would need to assign it to a cohort. 
+charlie = Cohort.first
+charlie.students.create name: "halie"
+=> Student id: 1, name: "halie", cohort_id: 1
+the foreign key would show up on the instances of the student model and would read `cohort_id:`
```

Comment:
The prompt states that the foreign key was never added to the Student model to begin with, so when creating a new instance of a student, Rails won't add the cohort_id like it usually does. In order to add a foreign key, we would need to make a new migration in order to add a new column to the Student model.